atlasdb-cassandra/build.gradle

```diff
@@ -1,55 +1,37 @@
-apply plugin: 'com.github.johnrengelman.shadow'
-
 apply from: "../gradle/shared.gradle"
+apply plugin: 'java-library'
```

Comment:
I _think_ this is already applied by `shared.gradle`, so there shouldn't be a need to re-specify this